src/frontend/app/components/ScenarioGrid.tsx

```diff
@@ -3,24 +3,41 @@
 import { MouseEvent, useCallback, useEffect, useRef } from 'react';
 import Link from 'next/link';
 import { useParams, useRouter, useSearchParams } from 'next/navigation';
-
 import { CategoryOptions, Scenario } from '@/interfaces/scenario';
 import BookmarkIcon from './BookmarkIcon';
 import TagList from './TagList';
 import { useSidebarContext } from '../context/sidebar';
 import { getElementPosition, isElementInView } from '@/helpers/ui';
+import CTABanner from './CTABanner';
 
 type ScenarioGridProps = {
   scenarios: Scenario[];
 };
 
+function splitItems(items: Scenario[]) {
```

Comment:
Just a suggestion, a comment here would make it easier to follow the function logic